azerothcore/azerothcore-wotlk
# DB/Quest: Ogre Head On A Stick = Party

-Simple to fix, but it was hard to find the fix

Co-Authored-By: WiZZy <bilucristian25@gmail.com>





Quest Ogre Head On A Stick = Party 

##### CHANGES PROPOSED:

-  Changed Faction 
-  


###### ISSUES ADDRESSED:

The issue was, as player you're not able to finish it. Actually not able to use quest item on that object. Just with .gm on it was posible.
Closes 


##### TESTS PERFORMED:

Tested in game, the players are able to finish it now. 
@wizzymore also tested it, everything fine :smiley: 


##### HOW TO TEST THE CHANGES:

1. .go gobject 4521
2. .quest add 3825 (You should be able to use quest that item on that object)


##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
